Update/omot_003.txt

```diff
@@ -415,7 +415,7 @@ void main()
 //r不愉快なのは、見透かされているように感じるからだ。『オヤシロさま』の名前を出しても、今はもう、私のそばにはいない――そのことを、指摘されているようで…。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "不愉快なのは、見透かされているように感じるからだ。『オヤシロさま』の名前を出しても、今はもう、私のそばにはいない——そのことを、指摘されているようで…。",
-			NULL, "I realized why I felt so unpleasant. I had mentioned the name of \"Oyashiro-sama,\" the one who was no longer by my side... I, myself, emphasized that point.", Line_Normal);
+			NULL, "The reason why I felt so unpleasant was that I felt like she was seeing through me. I had mentioned the name of \"Oyashiro-sama,\" the one who was no longer by my side... She looked like she was showing me that point...", Line_Normal);
```

Comment:
```suggestion
			NULL, "It was unpleasant because it felt like she was seeing through me. I had mentioned the name of \"Oyashiro-sama,\" the one who was no longer by my side... And it seemed like she was emphasizing that point...", Line_Normal);
```